Marlin/Configuration_adv.h

```diff
@@ -1561,11 +1561,21 @@
    * this setting determines the minimum update time between checks. A value of 100 works well with
    * error rolling average when attempting to correct only for skips and not for vibration.
    */
-  #define I2CPE_MIN_UPD_TIME_MS     100                     // Minimum time in miliseconds between encoder checks.
+  #define I2CPE_MIN_UPD_TIME_MS     4                       // Minimum time in miliseconds between encoder checks.
 
   // Use a rolling average to identify persistant errors that indicate skips, as opposed to vibration and noise.
   #define I2CPE_ERR_ROLLING_AVERAGE
 
+  // BABYSTEPPING and BABYSTEP_XY are required if ECM_MICROSTEP is used as a correction method
+  #if(I2CPE_ENC_1_EC_METHOD == I2CPE_ECM_MICROSTEP || I2CPE_ENC_2_EC_METHOD == I2CPE_ECM_MICROSTEP || I2CPE_ENC_3_EC_METHOD == I2CPE_ECM_MICROSTEP || I2CPE_ENC_4_EC_METHOD == I2CPE_ECM_MICROSTEP || I2CPE_ENC_5_EC_METHOD == I2CPE_ECM_MICROSTEP)
+    #ifndef BABYSTEPPING
+      #define BABYSTEPPING
+    #endif
+    #ifndef BABYSTEP_XY
+      #define BABYSTEP_XY
+    #endif
+  #endif
```

Comment:
Belongs in `Conditionals_LCD.h` or just make a `SanityCheck.h` entry.

Reply:
According to the beginning of Conditionals_LCD, it is for conditions that must be set before configuration_adv. It doesn't seem to make much sense to put this check there because the conditions it is based upon are set in configuration_adv. The other option you mention, but Sanity check isn't used for automatically enabling things is it? It would just be a user message to go enable it right?

Reply:
I think I figured it out. See comment on pull request.